Sources/LiveKit/Views/VideoView.swift

```diff
@@ -203,9 +201,18 @@ public class VideoView: NativeView, Loggable {
                 }
             }
 
-            // renderState updated
-            if state.renderState != oldState.renderState, let track = state.track {
-                track.notify { $0.track(track, videoView: self, didUpdate: state.renderState) }
+            // isRendering updated
+            if state.isRendering != oldState.isRendering, let track = state.track {
+
+                self.log("isRendering \(oldState.isRendering) -> \(state.isRendering)")
+
+                if state.isRendering {
+                    self._renderTimer.restart()
+                } else {
+                    self._renderTimer.suspend()
```

Comment:
Is this an irreversible thing? i. e. once `isRendering` goes to `false`, will it not change back and restart the timer?

Reply:
It's reversible because when a frame arrives state.isRendering will be true.